protobuf/bimrpc/bimrpc.proto

```diff
@@ -0,0 +1,115 @@
+syntax="proto3";
+
+package bimrpc;
+
+option go_package = "github.com/longhorn/types/pkg/bimrpc";
```

Comment:
```suggestion
option go_package = "github.com/longhorn/types/pkg/generated/bimrpcpb";
```

For Python generated module, suggest `generated-py/...` instead of `rpc-py`. WDYT?

Reply:
@innobead, If we want to update `go_package` in `bimrpc`, I will update all `go_package` with prefix `github.com/longhorn/types/pkg/generated` for consistency. The `generated-py` path also looks good to me.

Reply:
Sounds good.

Reply:
Updated it. Thank you.